src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/profiler/ProfilerEditingTarget.java

```diff
@@ -500,8 +531,13 @@ public void onError(ServerError error)
    
    private void persistHtmlPath(String htmlPath)
    {
+      persistDocumentProperty("htmlPath", htmlPath);
```

Comment:
@jjallaire yes, that sounds right; @javierluraschi I think that's the right spot. We could make them (and the viewer cache, for that matter) more scoped since technically these are bound to an RStudio "session", but I don't think it's necessary since the chances of collision are very low.